plugins/net_plugin/net_plugin.cpp

```diff
@@ -217,7 +217,7 @@ namespace eosio {
    public:
       unique_ptr<tcp::acceptor>        acceptor;
       ssl_context_ptr                  ssl_context;
-      bool                             ssl_enabled;
+      bool                             ssl_enabled{false};
```

Comment:
I suspect that an issue was how we initialize it:
```
   net_plugin::net_plugin()
      :my( new net_plugin_impl ) {
      my_impl = my.get();
   }
   ```
   `new net_plugin_impl{}` would fix it, isn't it?

Reply:
I'm not sure if that would fix it, but we do like to initialize primitive types explicitly and setting it explicitly to false fixes it.